signIn/signIn.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;500;600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./signIn.css">
+    <title>SignIn</title>
+</head>
+
+<body>
+    <main class="main-wrapper">
+        <div class="image-background">
+            <img class="img-header" src="../images/bg.jpg" alt="background images">
+        </div>
+
+        <form class="form-signup" id="form" submit="sendForm()">
+            <h2 class="name-page">Sign In</h2>           
+            <div class="wrapper-input">
+                <label for="email"></label>
+                <img class="img-icon" src="../images/icon-email.svg" alt="icon">
```

Comment:
Better to use background for these images. You won't interact with them.

Reply:
You can use pseudo-class ` ::before` for input  and set this svg icon like background-image for `::before`
Example: `
input {
position:relative;
}

input:before {
position:absolute;
top:0;
left:0; 
display:block;
width:24px;
height:24px;
content:'';
background-image:url(local path for svg icon);
}`